cmake/onnxruntime_providers.cmake

```diff
@@ -1535,6 +1535,12 @@ if (onnxruntime_USE_ROCM)
     target_compile_definitions(onnxruntime_providers_rocm PRIVATE ROCBLAS_BETA_FEATURES_API)
   endif()
 
+  if (onnxruntime_USE_HIPBLASLT)
+    find_package(hipblaslt)
```

Comment:
`find_package(hipblaslt REQUIRED)`, otherwise, cmake may silently pass the configure and produce compiler/linker error.